.github/workflows/send_slack_notifications.yaml

```diff
@@ -0,0 +1,24 @@
+name: Send Slack Notifications
+
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - opened
+
+jobs:
+  send_slack_notification:
+    name: Send New PR Slack notification
+    runs-on: ubuntu-latest
+    env:
+      PR_URL: 'https://github.com/deriv-com/quill-icons-park/pull/${{ github.event.pull_request.number }}'
+    steps:
```

Comment:
Can we add a message for successful or failed released? as an example please check the quill-icons release workflow